create-repeating-letters/main.js

```diff
@@ -0,0 +1,8 @@
+function createRepeatingLetters(word, numOfRepeats){
+  var repeatLetter = '';
+  for (var i = 0; i < word.length; i++){
```

Comment:
You have an extra loop here. You are returning right after the end of the interior loop, so this exterior doesn't even run more than one iteration :grin: